src/objectExplorer/objectExplorerService.ts

```diff
@@ -467,8 +467,11 @@ export class ObjectExplorerService {
             sessionId: node.sessionId,
             nodePath: node.nodePath
         };
-        await this._connectionManager.client.sendRequest(RefreshRequest.type, refreshParams);
-        return this._objectExplorerProvider.refresh(undefined);
+        let response = await this._connectionManager.client.sendRequest(RefreshRequest.type, refreshParams);
+        if (response) {
+            this._treeNodeToChildrenMap.delete(node);
```

Comment:
Why are you adding this? It's not clear how this is tied to your changes.

Reply:
It's not needed, but good to have